Instruments/config/2016/Skimmer.cfg

```diff
@@ -132,160 +134,99 @@ file: WJetsToLNu_HT-400To600.root
-file_ex: Radion_M-400.root GluGluToRadionToHHTo2B2Tau_M-400_narrow_ext1.root
-file_ex: Radion_M-450.root GluGluToRadionToHHTo2B2Tau_M-450_narrow.root
-file_ex: Radion_M-500.root GluGluToRadionToHHTo2B2Tau_M-500_narrow.root
-file_ex: Radion_M-550.root GluGluToRadionToHHTo2B2Tau_M-550_narrow.root
-file_ex: Radion_M-600.root GluGluToRadionToHHTo2B2Tau_M-600_narrow.root
-file_ex: Radion_M-650.root GluGluToRadionToHHTo2B2Tau_M-650_narrow.root
-file_ex: Radion_M-750.root GluGluToRadionToHHTo2B2Tau_M-750_narrow.root
-file_ex: Radion_M-800.root GluGluToRadionToHHTo2B2Tau_M-800_narrow.root
-file_ex: Radion_M-900.root GluGluToRadionToHHTo2B2Tau_M-900_narrow.root
+file_ex: Radion_M-250.root   GluGluToRadionToHHTo2B2Tau_M-250_narrow.root
+file_ex: Radion_M-260.root   GluGluToRadionToHHTo2B2Tau_M-260_narrow.root
+file_ex: Radion_M-270.root   GluGluToRadionToHHTo2B2Tau_M-270_narrow.root
+file_ex: Radion_M-280.root   GluGluToRadionToHHTo2B2Tau_M-280_narrow.root
+file_ex: Radion_M-300.root   GluGluToRadionToHHTo2B2Tau_M-300_narrow_ext1.root
+file_ex: Radion_M-320.root   GluGluToRadionToHHTo2B2Tau_M-320_narrow.root
+file_ex: Radion_M-340.root   GluGluToRadionToHHTo2B2Tau_M-340_narrow.root
+file_ex: Radion_M-350.root   GluGluToRadionToHHTo2B2Tau_M-350_narrow.root
+file_ex: Radion_M-400.root   GluGluToRadionToHHTo2B2Tau_M-400_narrow_ext1.root
+file_ex: Radion_M-450.root   GluGluToRadionToHHTo2B2Tau_M-450_narrow.root
+file_ex: Radion_M-500.root   GluGluToRadionToHHTo2B2Tau_M-500_narrow.root
+file_ex: Radion_M-550.root   GluGluToRadionToHHTo2B2Tau_M-550_narrow.root
+file_ex: Radion_M-600.root   GluGluToRadionToHHTo2B2Tau_M-600_narrow.root
+file_ex: Radion_M-650.root   GluGluToRadionToHHTo2B2Tau_M-650_narrow.root
+file_ex: Radion_M-800.root   GluGluToRadionToHHTo2B2Tau_M-800_narrow.root
+file_ex: Radion_M-900.root   GluGluToRadionToHHTo2B2Tau_M-900_narrow.root
 
 [Signal_Graviton]
 file_ex: Graviton_M-250.root GluGluToBulkGravitonToHHTo2B2Tau_M-250_narrow.root
 file_ex: Graviton_M-260.root GluGluToBulkGravitonToHHTo2B2Tau_M-260_narrow.root
 file_ex: Graviton_M-270.root GluGluToBulkGravitonToHHTo2B2Tau_M-270_narrow.root
 file_ex: Graviton_M-280.root GluGluToBulkGravitonToHHTo2B2Tau_M-280_narrow.root
-file_ex: Graviton_M-300.root GluGluToBulkGravitonToHHTo2B2Tau_M-300_narrow_ext1.root
+file_ex: Graviton_M-300.root GluGluToBulkGravitonToHHTo2B2Tau_M-300_narrow.root
 file_ex: Graviton_M-320.root GluGluToBulkGravitonToHHTo2B2Tau_M-320_narrow.root
 file_ex: Graviton_M-340.root GluGluToBulkGravitonToHHTo2B2Tau_M-340_narrow.root
 file_ex: Graviton_M-350.root GluGluToBulkGravitonToHHTo2B2Tau_M-350_narrow.root
 file_ex: Graviton_M-400.root GluGluToBulkGravitonToHHTo2B2Tau_M-400_narrow_ext1.root
 file_ex: Graviton_M-450.root GluGluToBulkGravitonToHHTo2B2Tau_M-450_narrow.root
-file_ex: Graviton_M-500.root GluGluToBulkGravitonToHHTo2B2Tau_M-500_narrow.root
+file_ex: Graviton_M-500.root GluGluToBulkGravitonToHHTo2B2Tau_M-500_narrowg.root
```

Comment:
if the name of the file is really "narrowg" -> rename it in the central storage to be "narrow", if it is a mistype -> correct it here.